x-pack/plugins/cases/public/components/connectors/fields_form.tsx

```diff
@@ -39,12 +37,7 @@ const ConnectorFieldsFormComponent: React.FC<Props> = ({ connector, isEdit, onCh
           }
         >
           <div data-test-subj={'connector-fields'}>
-            <FieldsComponent
-              isEdit={isEdit}
-              fields={fields}
-              connector={connector}
-              onChange={onChange}
-            />
+            <FieldsComponent connector={connector} key={connector.id} />
```

Comment:
The props are not needed anymore as a) the preview got separated from the form and b) the form lib handles the state of the form.